Makefile

```diff
@@ -43,7 +43,7 @@ INCPATH += -I$(FABRIC_PATH)/include
 endif
 
 ifeq ($(USE_UCX), 1)
-LIBS += -lucp -luct -lucs -lucm
+LIBS += -lucp -luct -lucs -lucm -lcudart
```

Comment:
Could you remove `-lcudart`? The CI only test build with USE_CUDA=0 and it causes the CI to fail